rest/utilities_testing.go

```diff
@@ -1968,3 +2067,18 @@ func (sc *ServerContext) suspendDatabase(t *testing.T, ctx context.Context, dbNa
 
 	return sc._suspendDatabase(ctx, dbName)
 }
+
+// getRESTkeyspace returns a keyspace for REST URIs
+func getRESTKeyspace(_ testing.TB, dbName string, collection *db.DatabaseCollection) string {
+	return strings.Join([]string{dbName, collection.ScopeName(), collection.Name()}, base.ScopeCollectionSeparator)
```

Comment:
Could this use collection.dbCtx to get the dbName, to avoid having to pass it in?